public/gameInfo.php

```diff
@@ -101,14 +124,30 @@
                 Are you sure that you want to view this game?
                 <br/>
                 <br/>
-                <form id='consentform' action='/game/<?= $gameID ?>' style='float:left'>
-                    <input type='hidden' name='v' value='1'/>
-                    <input type='submit' value='Yes. I&apos;m an adult'/>
-                </form>
-                <form id='escapeform' action='/gameList.php' style='float:left; margin-left:16px'>
-                    <input type='hidden' name='c' value='<?= $consoleID ?>'/>
-                    <input type='submit' value='Not Interested'/>
-                </form>
+
+                <div class="flex flex-col sm:flex-row gap-4 sm:gap-2">
+                    <form id='escapeform' action='/gameList.php'>
+                        <input type='hidden' name='c' value='<?= $consoleID ?>'/>
+                        <input type='submit' class='leading-normal' value='No. Get me out of here.'/>
+                    </form>
+
+                    <form id='consentform' action='/game/<?= $gameID ?>'>
+                        <input type='hidden' name='v' value='1'/>
+                        <input type='submit' class='leading-normal' value='Yes. I&apos;m an adult.'/>
+                    </form>
+
+                    <?php if ($userWebsitePrefs): ?>
+                        <button 
+                            class='break-words whitespace-normal leading-normal' 
+                            onclick='disableMatureContentWarningPreference(
+                                <?= $userWebsitePrefs ?>,
+                                <?= $gameID ?>
+                            )'
+                        >
+                            Yes. Never ask me again for games with mature content.
```

Comment:
Going back to my [previously referenced web page](https://www.margieholdscourt.com/after-yes-and-no/#:~:text=Let's%20keep%20it%20simple%3A,the%20question%2C%20use%20a%20comma.), I believe it's acceptable to say "Yes. And never..."

Reply:
Done